Mathlib/RingTheory/Coalgebra/Basic.lean

```diff
@@ -181,6 +192,8 @@ theorem comul_apply (r : R) : comul r = 1 ⊗ₜ[R] r := rfl
 @[simp]
 theorem counit_apply (r : R) : counit r = r := rfl
 
+instance : IsCocomm R R where comul_comm' := by ext; simp
```

Comment:
Are there any other interesting instances you could add in this PR?

Reply:
How easy is the Finsupp instance to define in this PR? I assume it lifts for free to monoidalgebra via defeq abuse?

Reply:
Easy